apps/templates/home/welcome.html

```diff
@@ -66,6 +66,12 @@ <h1>WELCOME TO DEEP'S API ENDPOINT</h1>
         >
             <image class="icon" alt="ReDoc" src="{% static 'drf-yasg/redoc/redoc-logo.png' %}"/>
         </a>
+        <a
+        class="icon"
+        href="/graphql-docs/"
+    >
+        <image class="icon" alt="graphqlDocs" src="{% static 'image/graphQL_logo' %}"/>
```

Comment:
Related to: https://github.com/the-deep/server/pull/1325#issuecomment-1596743203
Missing extension here.
```suggestion
        <image class="icon" alt="graphqlDocs" src="{% static 'image/graphQL_logo.svg' %}"/>
```

Can you also confirm and add a snapshot in the PR description of what the new welcome page looks like?